test/images/agnhost/netexec/netexec.go

```diff
@@ -192,7 +199,6 @@ func addRoutes(mux *http.ServeMux, exitCh chan shutdownRequest) {
 }
 
 func startServer(server *http.Server, exitCh chan shutdownRequest, fn func() error) {
-	log.Printf("Started HTTP server on port %d", httpPort)
```

Comment:
any special reason to remove it?
actually, I think we should add logging too to the udp servers when starting

Reply:
duh, it does have logging, forget it, just enable this one if there is no special reason to remove it

Reply:
My bad, must have been for when I tried to extend the multi-address to other protocols.
Bringing it back.